src/components/Tile/tile.scss

```diff
@@ -0,0 +1,176 @@
+//===================
+//SMALL TILE STYLES
+//===================
+.tile--sm {
+  display: flex;
+  flex-direction: column;
+
+  & > .tile--text-container {
+    padding-left: 0;
+
+    & > p {
+      font-size: calc(14px + 2 * ((100vw - 320px) / 1264));
+    }
+  }
+
+  @include max-breakpoint('lg'){
+    margin-bottom: rem(20px);
+  }
+}
+
+.title--sm {
+  margin-bottom: rem(14px);
+  @include type-style('productive-heading-04');
+  font-size: calc(20px + 8 * ((100vw - 320px) / 1264));
+}
+
+.img--sm {
+  width: 100%;
+  height: auto;
+  margin-bottom: rem(18px);
+
+  & > .gatsby-resp-image-wrapper {
+    max-width: rem(500px) !important;
+  }
+}
+
+.arrow--link-container {
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+
+  & > a {
+    padding-bottom: rem(4px);
+    padding-right: rem(4px);
+  }
+
+  & > svg {
+    fill: $brand-01;
+  }
+}
+
+//===================
+//LARGE TILE STYLES
+//===================
+
+.tile--lg, .tile--xl {
+  display: flex;
+  margin-bottom: rem(30px);
+  justify-content: space-between;
+
+  & > .tile--text-container {
+    margin-top: rem(80px);
+
+    & > h1,
+    & > p,
+    & > div {
+      padding-left: rem(16px);
+    }
+
+    & > p {
+      line-height: rem(22px);
+      font-size: calc(14px + 2 * ((100vw - 320px) / 1264));
+    }
+
+    @include max-breakpoint('lg'){
+      margin-left: rem(20px);
+      margin-right: rem(20px);
+    }
+
+    @include max-breakpoint('bp--xs--major'){
+      margin-top: rem(40px);
+    }
+  }
+
+  @include max-breakpoint('lg'){
+    flex-direction: column;
+  }
+}
+
+.title--lg {
+  font-size: calc(24px + 18 * ((100vw - 320px) / 1264));
+  color: $brand-01;
+  font-weight: 400;
+}
+
+.tile--button.bx--btn {
+  display: flex;
+  justify-content: space-between;
+  width: rem(208px);
+  height: rem(65px);
+  padding-right: rem(16px);
+  margin-top: rem(42px);
+
+  @include max-breakpoint('lg'){
+    margin-top: rem(20px);
+  }
+
+  & > svg {
+    fill: $ibm-colors__white;
+  }
+}
+
+.title--two {
+  font-weight: 400;
+  font-size: calc(24px + 18 * ((100vw - 320px) / 1264));
+  color: $ibm-colors__black;
+  margin-bottom: rem(30px);
+
+  @include max-breakpoint('lg'){
+    margin-bottom: rem(20px);
+  }
+}
+
+.img--lg {
+  margin-right: rem(65px);
+  margin-top: rem(172px);
+  margin-bottom: rem(72px);
+
+  @include max-breakpoint('lg'){
+    margin-top: rem(50px);
+  }
+}
+
+.tile--xl {
+  flex-direction: column;
+
+  & > .tile--text-container {
+    max-width: rem(640px);
```

Comment:
after unnesting all the grid stuff, I had to make the `xl` tile `flex-direction: column` so that the image would be below the text, but then the text would span the whole width, (was ignoring `col-lg-8`) so I added this `max-width` just to follow design spec.